docs/website/docs/dlt-ecosystem/destinations/duckdb.md

```diff
@@ -102,29 +102,41 @@ p = dlt.pipeline(
 )
 ```
 
-The destination accepts a `duckdb` connection instance via `credentials`, so you can also open a database connection yourself and pass it to `dlt` to use. `:memory:` databases are supported.
+The destination accepts a `duckdb` connection instance via `credentials`, so you can also open a database connection yourself and pass it to `dlt` to use.
+
 ```py
 import duckdb
 db = duckdb.connect()
 p = dlt.pipeline(
-  pipeline_name='chess',
+  pipeline_name="chess",
   destination=dlt.destinations.duckdb(db),
-  dataset_name='chess_data',
+  dataset_name="chess_data",
   full_refresh=False,
 )
+
+# Or if you would like to use in-memory duckdb instance
+db = duckdb.connect(":memory:")
```

Comment:
imho
    p = dlt.pipeline(
        pipeline_name=uniq_id(), destination="duckdb", credentials=duckdb.connect(":memory:")
    )
looks a bit nicer

Reply:
Updated docs snippet to use `Destination.from_reference(...)` and added an additional check for it in the tests.